parser-typechecker/src/Unison/Codebase/Branch.hs

```diff
@@ -485,8 +518,10 @@ sync
   -> (EditHash -> m Patch -> m ())
   -> Branch m
   -> m ()
-sync exists serializeRaw serializeEdits b =
-  State.evalStateT (sync' exists serializeRaw serializeEdits b) mempty
+sync exists serializeRaw serializeEdits b = do
+  _written <- State.execStateT (sync' exists serializeRaw serializeEdits b) mempty
+  -- traceM $ "Branch.sync wrote " <> show (Set.size written) <> " namespace files."
+  pure ()
```

Comment:
Added some logging here, then commented it out again. Considered removing it entirely, but seems okay and kind of helpful as a reminder of how to print out what was sync'd.